src/praisonai-agents/praisonaiagents/agent/execution_mixin.py

```
@@ -28,6 +28,36 @@
+                app = _shared_apps.get(port)
+                if app is not None:
+                    try:
+                        app.router.routes = [
+                            r for r in app.router.routes
+                            if getattr(r, "path", None) != p
+                        ]
+                        # Invalidate cached OpenAPI schema so removed routes
+                        # disappear from /openapi.json and /docs.
+                        app.openapi_schema = None
+                    except Exception:
+                        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)